pkgs/development/python-modules/biliass/default.nix

```diff
@@ -3,28 +3,30 @@
   buildPythonPackage,
   fetchFromGitHub,
   pythonOlder,
-  poetry-core,
-  protobuf,
+  hatchling,
+  protobuf5,
   pytestCheckHook,
 }:
 
 buildPythonPackage rec {
   pname = "biliass";
-  version = "1.3.11";
+  version = "1.3.13";
   format = "pyproject";
 
   disabled = pythonOlder "3.7";
 
```

Comment:
```suggestion
```
I recommend you to drop this as it is not that useful and requires to be kept up to date.